packages/release-trigger/src/bot.ts

```diff
@@ -112,7 +120,8 @@ async function doTrigger(
   pullRequest: PullRequest,
   token: string,
   logger: GCFLogger,
-  installationId: number
+  installationId: number,
```

Comment:
Add @params to the function's doc comment for installationId and multiScmName.

Reply:
Done.